client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -1,37 +1,64 @@
-import React, { useEffect, useContext } from 'react';
-import { useSelector } from 'react-redux';
-import StoreStateType from 'redux/storeStateType';
-import { Collapse, Divider, Typography } from '@material-ui/core';
 import Swal from 'sweetalert2';
+import { useSelector } from 'react-redux';
+import { AddCircle } from '@material-ui/icons';
+import React, { useEffect, useContext, useState } from 'react';
+import { Collapse, Divider, Typography, IconButton } from '@material-ui/core';
 
 import axios from 'Utils/axios';
 import Toggle from 'commons/Toggle/Toggle';
+import StoreStateType from 'redux/storeStateType';
 import FormRowWithInput from 'commons/FormRowWithInput/FormRowWithInput';
-import { exposureAndFlightsContext, fieldsNames } from 'commons/Contexts/ExposuresAndFlights';
+import { exposureAndFlightsContext, fieldsNames, Exposure, initialExposureOrFlight, isConfirmedExposureInvalid, isFlightInvalid } from 'commons/Contexts/ExposuresAndFlights';
 
-import FlightsForm from './FlightsForm/FlightsForm';
-import ExposureForm from './ExposureForm/ExposureForm';
 import useFormStyles from 'styles/formStyles';
+import FlightsForm from './FlightsForm/FlightsForm';
 import useStyles from './ExposuresAndFlightsStyles';
+import ExposureForm from './ExposureForm/ExposureForm';
 
+const addConfirmedExposureButton: string = 'הוסף חשיפה';
+const addFlightButton: string = 'הוסף טיסה לחול';
 
 const ExposuresAndFlights = () => {
-  const context = useContext(exposureAndFlightsContext);
-  const { exposureAndFlightsData, setExposureDataAndFlights } = context;
+  const { exposureAndFlightsData, setExposureDataAndFlights } = useContext(exposureAndFlightsContext);;
+  const { exposures } = exposureAndFlightsData;
 
   const investigationId = useSelector<StoreStateType, number>((state) => state.investigation.epidemiologyNumber);
 
   const { fieldName } = useFormStyles();
   const classes = useStyles();
 
+  const disableConfirmedExposureAddition : boolean= React.useMemo(() => 
+    exposures.some(exposure => exposure.wasConfirmedExposure && isConfirmedExposureInvalid(exposure))
```

Comment:
this condition checks if some of the exposures are invalid - but then if one of the exposures in the db is invalid i cant add a new one.
is this the requested behaviour?

Reply:
it just isnt really clear in the ui - because by default I cant add an exposure if an exposure exists with incomplete data. 
more of question to Tal, but maybe separate the UI of a new exposure form vs existing exposure or add message / tooltip on button.
Anyway, if this behaviour is expected those questions/suggestions aren't urgent

Reply:
Well you are not supposed to have invalid exposures on db, and if you do I really think adding new forms when the first are invalid isn't good. Anyway, Elad's team will change the validations when this merged to their standarts, so it might change

Reply:
I mean, we do allow them in the db 🤷‍♀️  but yes it does seem more relevant to @eladyashar